examples/01-getting-started.ipynb

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
<p><code>Merlin Models is a library and part of the Merlin open source frameowrk</code> </p><p> </p><p>Can we rework this sentence? Saying MM is a library doesn't say much - how about merging with the 2nd sentence, and delay mentioning nvidia merlin to the end of the paragraph. </p><p> </p><pre class="ql-syntax" spellcheck="false">Merlin Models is a library for training and deploying recommender models. Merlin Models let susers in industry easily train standard models against their own dataset, getting high performance GPU accelerated models into production with best practices baked into the library. This will also let researchers to build custom models by incorporating standard components of deep learning recommender models, and then benchmark their new models on example offline datasets. Merlin Models is part of the Merlin open source framework

</pre><p> </p><p>(also framework is mispelled )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I updated the text. I removed the references to production. Merlin Models is for training recommender systems. I think Merlin Systems is to get them into production?